src/main/rx/rx.h

```diff
@@ -129,6 +130,7 @@ typedef struct rxConfig_s {
     ioTag_t spektrum_bind_plug_ioTag;
     uint8_t spektrum_sat_bind;              // number of bind pulses for Spektrum satellite receivers
     uint8_t spektrum_sat_bind_autoreset;    // whenever we will reset (exit) binding mode after hard reboot
+    uint8_t rssi_src_frame_errors;			// true to use frame drop flags in the rx protocol
```

Comment:
This struct is used in a parameter group. As such, all new parameters have to be appended at the end. Please move this.

Reply:
Done.

It's worth noting that in discussing filtering, the OSD applies more filtering than the FC.